pkg/providers/property/data_akamai_search_test.go

```diff
@@ -0,0 +1,70 @@
+package property
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/akamai/AkamaiOPEN-edgegrid-golang/v2/pkg/papi"
+)
+
+func TestDSSearch(t *testing.T) {
```

Comment:
This test failed with output:
```
=== RUN   TestDSSearch
--- FAIL: TestDSSearch (28.82s)
=== RUN   TestDSSearch/match_by_hostname
2022/03/02 01:29:29 [INFO]  Provider version: dev: OperationID=d582843e-6767-4b04-9a34-7d3b3aac6399
2022/03/02 01:29:30 [INFO]  Provider version: dev: OperationID=94ccc88f-d615-46d2-990f-bffcc922bc5e
    data_akamai_search_test.go:56: Step 1/1 error: Check failed: 1 error occurred:
        	* Check 2/2 error: data.akamai_search.test: Attribute 'properties' expected to be set
        
    --- FAIL: TestDSSearch/match_by_hostname (28.82s)


FAIL
```

Reply:
Test should be fixed now, thanks

Reply:
it works now ^_^

I have also realized that you added the failing unit test, amazing!